R/msgmerge.R

```diff
@@ -5,6 +5,7 @@
 run_msgmerge <- function(po_file, pot_file, previous = FALSE, verbose = TRUE) {
   args <- c(
     "--update", shQuote(path.expand(po_file)),
+    "--backup=off",
```

Comment:
ahhh nice! great find

Reply:
maybe we should check if a .git folder exists in TLD first, otherwise leave this on? or else expose this as an option defaulting to off?

Reply:
I personally don't think it's worth the effort just because pretty much everyone is going to be using git anyway. Automatically creating backups here might also imply that potools should also backup `.pot`.

Reply:
ok agreed. we can wait for such a feature request first at least.